proto_generator/protogenerator.go

```diff
@@ -115,7 +116,8 @@ func main() {
 		},
 		PackageName: *packageName,
 		Caller:      *callerName,
-		ProtoOptions: ygen.ProtoOpts{
+	},
+		&protogen.ProtoOpts{
```

Comment:
`protogen.Opts`.

Reply:
Proposal to address this here: https://github.com/openconfig/ygot/pull/706 PTAL